packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -35,22 +39,83 @@ export class SidebarComponent extends BaseContainer {
 	 */
 	async _postInit(): Promise< void > {
 		await this.page.waitForLoadState( 'networkidle' );
+		this.sidebar = await this.page.waitForSelector( selectors.sidebar );
```

Comment:
This statement will ask the `page` to confirm presence of the sidebar element, then return a handle to the element.

I have chosen to retain reference to this element to automatically narrow the scope of any selector related tasks to just the sidebar.